.env.prod

```diff
@@ -0,0 +1 @@
+ELASTICSEARCH_DSL__default__hosts=https://your-prod-es-host:9200
```

Comment:
Consider making this value configurable via deployment secrets or environment variables in your orchestration system, not hardcoded in the repo.